src/js/playlist/loader.js

```diff
@@ -65,9 +65,8 @@ const PlaylistLoader = function() {
 
     function playlistError(error) {
         if (!error.code) {
-            error = new PlayerError(error ? error : 'Error loading file', 0);
+            error = new PlayerError(MSG_CANT_LOAD_PLAYER, 0);
```

Comment:
I think `cantPlayVideo` would be a better message because the problem is the media files